README.md

```diff
@@ -612,6 +614,21 @@ SAGEMAKER_TFS_NUM_BATCH_THREADS="16"
 SAGEMAKER_TFS_MAX_ENQUEUED_BATCHES="10000"
 ```
 
+## Other Configurable Environment Variables
+The following environment variables can be set on a SageMaker Model or Transform Job if further configuration is required:
+
+```bash
+# Configures the logging level for GUnicorn.
+# Valid values can be found here: https://docs.gunicorn.org/en/stable/settings.html#loglevel
+# Defaults to "info"
+SAGEMAKER_GUNICORN_LOGLEVEL="debug"
```

Comment:
Add reference link here https://docs.gunicorn.org/en/stable/settings.html#loglevel
See my below comment for more details.